app/config/custom-environment-variables.json

```diff
@@ -8,6 +8,7 @@
   "keycloak": {
     "clientId": "KC_CLIENTID",
     "clientSecret": "KC_CLIENTSECRET",
+    "publicKey": "SERVER_KC_PUBLICKEY",
```

Comment:
This needs to be `KC_PUBLICKEY` as environment variable

Reply:
ok. done